src/dataflow-types/src/client/controller/storage.rs

```diff
@@ -493,6 +493,7 @@ where
         Ok(())
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
```

Comment:
I think all these want to be skip_all, the fn args would be pretty expensive to serialize